wgpu/examples/framework.rs

```diff
@@ -1,11 +1,14 @@
-use std::future::Future;
 #[cfg(not(target_arch = "wasm32"))]
 use std::time::{Duration, Instant};
+use std::{future::Future};
 use winit::{
     event::{self, WindowEvent},
     event_loop::{ControlFlow, EventLoop},
 };
 
+#[path = "../tests/common/mod.rs"]
```

Comment:
can we add `#[cfg(test)]` on top?

Reply:
Oh yes we should, that should also fix wasm